Example/Backpack Screenshot/SwiftUIScreenshots.swift

```diff
@@ -327,6 +327,11 @@ class SwiftUIScreenshots: BackpackSnapshotTestCase {
             saveScreenshot(component: "carousel-card", scenario: "default", userInterfaceStyle: userInterfaceStyle)
         }
         
+        await navigate(title: "Card Carousel") {
```

Comment:
I think this one is duplicated

Reply:
I think all components have two parts in this file 🙂